exercises/space-age/description.md

```diff
@@ -1,4 +1,4 @@
-Given an age in seconds, calculate how old someone would be on:
+ Given an age in seconds, calculate that age in terms of years of the different planets. Here is a list of the length of one 'year' on each planet:
```

Comment:
Just a nit pick, but it appears like there is a leading space that isn't needed.  Also, `"that"` sounds strange to me.  How about `a persons`?  It isn't clear what "that" is referring to.  Is it meant to be the age of the planet, or in the context of the original description, a persons age on a given planet?

Reply:
Both sound a little odd to me!
Maybe a more concise version would be
`Given a person's age in seconds, calculate their age in terms of years of the different planets. ...`

Reply:
I agree that what you have written is clearer